lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua

```diff
@@ -0,0 +1,9 @@
+return {
+  "OXY2DEV/markview.nvim",
+  ft = { "markdown", "markdown.mdx" },
+  dependencies = { "nvim-treesitter/nvim-treesitter" },
```

Comment:
on their docs, they also have `nvim-tree/nvim-web-devicons` as a dep, should we include this here too?

Reply:
It's fine without it because it will load on module

Reply:
sweet